### PR TITLE
Update GettingStartedWithTheMRTK.md

### PR DESCRIPTION
**Pull request by RogPodge**
## Overview
Adds helpful notice in the tutorial to lower the gizmos size for the HandInteractionExamples.unity scene
Gizmos

### CODE REVIEW

File: Documentation/GettingStartedWithTheMRTK.md [hunk 1]
```diff
@@ -146,6 +146,10 @@ To try the hand interaction scene, do the following steps.
 
 1. Close the TMP dialog. After this you need to reload the scene. You can do this by double-clicking the scene in the Project tab.
 
+1. Uncheck or shrink the size of the 3d icons under the Gizmos tab in the Scene view to make the scene easier to look at
+
+  [Insert image here]
```
Comment (RogPodge) [inline Documentation/GettingStartedWithTheMRTK.md:151]:
```suggestion
  ![Gizmos](https://user-images.githubusercontent.com/39840334/80771381-2978c000-8b08-11ea-8fd0-149b0c06fc6a.png)
```

File: Documentation/GettingStartedWithTheMRTK.md [hunk 2]
```diff
@@ -146,6 +146,10 @@ To try the hand interaction scene, do the following steps.
 
 1. Close the TMP dialog. After this you need to reload the scene. You can do this by double-clicking the scene in the Project tab.
 
+1. Uncheck or shrink the size of the 3d icons under the Gizmos tab in the Scene view to make the scene easier to look at
+
+  ![Gizmos](https://user-images.githubusercontent.com/39840334/80771381-2978c000-8b08-11ea-8fd0-149b0c06fc6a.png)
```
Comment (RogPodge) [inline Documentation/GettingStartedWithTheMRTK.md:151]:
```suggestion
     ![Gizmos](https://user-images.githubusercontent.com/39840334/80771381-2978c000-8b08-11ea-8fd0-149b0c06fc6a.png)
```

**Comment by RogPodge** [comment]
QOL documentation update

**Comment by keveleigh** [comment]
>Shall we use light gray theme editor since other screenshots on the same page are using light theme?

Ah, good point! I just brought this up in stand-up on light vs dark theme. I think the general consensus was to use the light theme, since not all contributors to the Toolkit might have a pro license and thus access to the dark theme, and we'd like to keep things as consistent as possible.
@RogPodge if possible, could you replace the image with a light theme one and add a comment in [our images / screenshots guide in the docs](https://microsoft.github.io/MixedRealityToolkit-Unity/Documentation/Contributing/DocumentationGuide.html#images--screenshots) to recommend using the light theme? Thanks!

**Comment by RogPodge** [comment]
Messed up by not merging this PR from my own branch. Closing this one and using https://github.com/microsoft/MixedRealityToolkit-Unity/pull/7762

**Pull request closed without merging**